src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelper.java

```diff
@@ -167,13 +178,22 @@ private static MultipleType getMultipleTypeFromCaseData(CaseData caseData) {
     }
 
     private static String getSubmitEventState(SubmitEvent submitEvent) {
-        return submitEvent.getState() != null
-                ? (submitEvent.getState().equals(PENDING_STATE) ? SUBMITTED_STATE : submitEvent.getState())
-                : " ";
+        if (submitEvent.getState() != null) {
```

Comment:
Definitely a good call to refactor that confusing code. I think it can still be improved a bit as submitEvent.getState() is called three times.

```
  private static String getSubmitEventState(SubmitEvent submitEvent) {
      String state = submitEvent.getState();
      if (state != null) {
          return state.equals(PENDING_STATE) ? SUBMITTED_STATE : state;
      } else {
          return " ";
      }
  }
```

Reply:
done